subprojects/docs/src/samples/plugins/publishing/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+// tag::complete-plugin-publishing[]
```

Comment:
:o: Doesn't seem like we need a tag here unless we intend to add a license here.